<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWin</class>
 <widget class="QWidget" name="SettingsWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_save_username">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>64</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_new_username">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>44</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>New username:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ent_new_username">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>64</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ent_new_password">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>124</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_new_password">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>104</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>New password:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save_password">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>124</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_theme">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>174</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Theme:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_themes">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>170</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Standard</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Light Sky</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Night</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_accept_theme">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>171</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_OK">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
